### ISSUE TITLE
Integration Firebase plugin is cause issues with another wordpress plugin

### ISSUE THREAD

**Issue by elmcapp** [dalenguyen/firebase-wordpress-plugin#42]
I am using integration firebase pro in my wordpress site. I am also using another plugin called ARMember which is a membership plugin that allow me to have membership plans. The issue is when I activate the integration firebase pro plugin it causes one of my pages from ARMember not to load

![see 2](https://user-images.githubusercontent.com/64275658/81588902-00b5bd80-9387-11ea-85d7-5f04f3a0d3b3.png)
![see](https://user-images.githubusercontent.com/64275658/81588904-014e5400-9387-11ea-8439-3bfb81c04a15.png)

**Comment by dalenguyen** [comment]
It seems that there’s an issue with the Datatable on that page. Does it work if you deactivate the plugin?

**Comment by elmcapp** [comment]
Yes. If I disable intergration firebase pro then it works. 

![order](https://user-images.githubusercontent.com/64275658/81628600-1a7af300-93cf-11ea-85f4-2f42e5c87c31.png)
Things I have tried: I download a plugin called plugin orgizer. This allow me to order plugins in the order that I want the to trigger (activate or be since on the site). I put Your plugin as number 1 and the ARMember plugin as number 2. When I did that everything seem to work. No sure if its a Datatable issue or .js issue
